package.json

```diff
@@ -3,8 +3,13 @@
   "version": "0.2.3",
   "description": "UKHSA Spectral ruleset",
   "main": ".spectral.yaml",
+  "type": "module",
   "scripts": {
-    "prepare": "husky && git config commit.template .gitmessage"
+    "prepare": "husky && git config commit.template .gitmessage",
+    "build": "tsc && npm run copy-legacy",
+    "copy-legacy": "cpx \"src/functions/legacy/*.js\" dist/functions",
+    "test": "NODE_OPTIONS='--experimental-vm-modules' jest --coverage",
```

Comment:
@collinsukhsa  is the --experimental-vm-modules still required given  the use of typescript